pandas/_libs/lib.pyx

```diff
@@ -1951,7 +1953,7 @@ def maybe_convert_numeric(ndarray[object] values, set na_values,
 @cython.wraparound(False)
 def maybe_convert_objects(ndarray[object] objects, bint try_float=0,
                           bint safe=0, bint convert_datetime=0,
-                          bint convert_timedelta=0):
+                          bint convert_timedelta=0, to_integer_array=False):
```

Comment:
ok with a keyword (here as this is in internal funtion), but can you name
```bint convert_to_nullable_integer=False```

Reply:
Variable name refactored.